### PR TITLE
Fetch SSH keys from GitHub

### PR DESCRIPTION
**Pull request by makyo**
Adds the ability to, down the line, fetch SSH keys from GitHub using the public API.

### CODE REVIEW

**Comment by makyo** [comment]
# QA

With the GUI running behind guiproxy, open the dev tools and type `jujugui.sshKeys.githubSSHKeys(new yui.juju.environments.web.WebHandler, 'makyo', console.log)` at the console and see my four keys fetched and parsed. Type `jujugui.sshKeys.githubSSHKeys(new yui.juju.environments.web.WebHandler, 'makyoa', console.log)` and see an error returned.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7850/

**Comment by makyo** [comment]
Thanks all :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

File: jujugui/static/gui/src/app/utils/github-ssh-keys.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+/**
+  Fetch SSH keys from a user's GitHub account.
+
+  @param username {String} the user whose keys need fetching.
+  @param callback {Function} a function expecting an error and the parsed keys.
+*/
+const githubSSHKeys = (handler, username, callback) => {
+  // On load, call the callback with munged data.
+  const wrap = (response) => {
+    let data = JSON.parse(response.currentTarget.response);
```
Comment (hatched) [inline jujugui/static/gui/src/app/utils/github-ssh-keys.js]: Because we don't control the response can you add a try/catch here please.

Comment (Lukewh) [inline jujugui/static/gui/src/app/utils/github-ssh-keys.js]: I can do this as part of the branch I'm working on @hatched @makyo

**Pull request merged**